roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -201,9 +246,11 @@ class Namespace(object):
 
 
 class ResetNeutron(object):
-    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-linuxbridge-agent']
+    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-agent', 'neutron-linuxbridge-agent']
 
     def run(self):
+        nova_iface = Interface.get_nova_interface()
```

Comment:
please use plural variable name and method names for ones which contain/return enumerable.